src/ucp/tag/tag_send.c

```diff
@@ -276,6 +276,7 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_nbx,
         goto out;
     });
 
+    ucp_request_send_state_clear(req, NULL);
```

Comment:
i'm pretty sure it's initialized later on already..

Reply:
same - failed valgrind check in CI